### PR TITLE
favicons update

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/opensourcedesign/opensourcedesign.github.io/issues/353

### CODE REVIEW

**Comment by Arhell** [comment]
cc @Erioldoesdesign 
@ZoranPandovski

File: images/favicons/safari-pinned-tab.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (RDIL) [inline images/favicons/safari-pinned-tab.svg:1]: Ideally this should be compressed to reduce loading times.

**Review by RDIL — CHANGES_REQUESTED** [review]

File: images/favicons/safari-pinned-tab.svg (hunk 1 above)
Comment (Arhell) [reply images/favicons/safari-pinned-tab.svg:1]: Ok

**Comment by Arhell** [comment]
@RDIL done

**Comment by Arhell** [comment]
cc @Erioldoesdesign

**Review by RDIL — APPROVED** [review]
LGTM

**Pull request merged**